RefRed/calculations/check_list_run_compatibility_thread.py

```diff
@@ -107,10 +107,26 @@ def update_lconfigdataset(self):
         self.parent.big_table_data = big_table_data
 
     def loading_lr_data(self):
+        """Updates data table with the new run."""
         wks = self.wks
         lrdata = LRData(wks, parent=self.parent)
         self.lrdata = lrdata
         big_table_data = self.parent.big_table_data
         col_index = 0 if self.is_working_with_data_column else 1
+        # keep existing user config if the new run number is the same as the old one
+        if self.is_reloading_same_run():
+            lrdata.peak = big_table_data[self.row, col_index].peak
+            lrdata.back = big_table_data[self.row, col_index].back
+            lrdata.low_res = big_table_data[self.row, col_index].low_res
         big_table_data[self.row, col_index] = lrdata
         self.parent.big_table_data = big_table_data
```

Comment:
why do we need this assignment? Line 114 states both `big_table_data` and `self.parent.big_table_data` point to the same memory address, right? :thinking: